build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
     id 'pmd'
 }
 
-group 'be.xplore'
+group 'com.github'
```

Comment:
This must be com.github.autobump. The group must also be unique when publishing artifacts to Maven Central.